public/scripts/frames.js

```diff
@@ -198,4 +199,20 @@ function fillDataIntoLogWorkPage(id, title, estimatedTime, logWorkTotalTime, wor
     document.getElementById("logWorkTotalTime").innerHTML = logWorkTotalTime;
     document.getElementById("logWorkDone").value = workDone;
     document.getElementById("logWorkRT").innerHTML = estimatedTime - logWorkTotalTime;
+}
+
+function fillDataIntoEditPageTask(id, title, des, estimatedTime, difficulty, priority, assignee, creator, status, category, sprintNumber) {
+    console.log(title);
```

Comment:
remove the console.log(title);  from here.
It was written for checking purposes.

Reply:
removed